static/js/utils/disaster_event_map_utils.tsx

```diff
@@ -523,12 +523,12 @@ export function getSeverityFilters(
 }
 
 /**
- * Gets whether or not to use cache from url params.
+ * Gets whether or not to use cache for data fetch.
```

Comment:
nit: maybe mention that the cache here is JSON, to link the two.

Reply:
updated the comment